data/shaders/globals.glsl

```diff
@@ -0,0 +1,6 @@
+uniform b_Globals {
+    mat4 u_ViewProj;
+    mat4 u_InverseProj;
```

Comment:
the name is incorrect. Should be something like `u_InverseRotateProj` to reflect the following facts:
  1. it does include the view transform
  2. but not the translation portion of it...

Reply:
Actually it's inverted camera projection matrix, it's not View matrix.

Reply:
It's not **just** the inverted projection, that's my concern